flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/wrappers/StormTuple.java

```diff
@@ -44,6 +45,21 @@
 	/** The schema (ie, ordered field names) of the tuple */
 	private final Fields schema;
 
+	private final int taskId;
+	private final String streamId;
+	private final MessageId id;
+	private final String componentId;
+
```

Comment:
I have already worked on this and I think my solution is smoother as it compute all this stuff automatically under the hood (without the need that the user specifies the name in the constructor). No PR yet as not finished completely. Need to get in sync about it.